Formula/phpunit@7.5.rb

```diff
@@ -0,0 +1,112 @@
+class PhpunitAT75 < Formula
```

Comment:
Versioned formula should only depend on major version differences. This should be just 7.